### PR TITLE
fix(plasma-tokens): Update token template for Kotlin and React-Native platforms

### PR DESCRIPTION
**Pull request by neretin-trike**
:baby_chick: Download canary assets:

[colors--canary.305.4053370533.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.305.4053370533.xml)  
[color_metro_ios-swift--canary.305.4053370533.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.305.4053370533.swift)  
[color_metro_kotlin--canary.305.4053370533.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.305.4053370533.kt)  
[color_metro_react-native--canary.305.4053370533.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.305.4053370533.ts)  
[color_sbermarket_ios-swift--canary.305.4053370533.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.305.4053370533.swift)  
[color_sbermarket_kotlin--canary.305.4053370533.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.305.4053370533.kt)  
[color_sbermarket_react-native--canary.305.4053370533.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.305.4053370533.ts)  
[color_sberprime_ios-swift--canary.305.4053370533.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.305.4053370533.swift)  
[color_sberprime_kotlin--canary.305.4053370533.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.305.4053370533.kt)  
[color_sberprime_react-native--canary.305.4053370533.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.305.4053370533.ts)  
[color_selgros_ios-swift--canary.305.4053370533.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.305.4053370533.swift)  
[color_selgros_kotlin--canary.305.4053370533.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.305.4053370533.kt)  
[color_selgros_react-native--canary.305.4053370533.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.305.4053370533.ts)  
[color_smbusiness_ios-swift--canary.305.4053370533.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.305.4053370533.swift)  
[color_smbusiness_kotlin--canary.305.4053370533.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.305.4053370533.kt)  
[color_smbusiness_react-native--canary.305.4053370533.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.305.4053370533.ts)  
[PlasmaTokensColor--canary.305.4053370533.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.305.4053370533.swift)  
[typo_mage_ios-swift--canary.305.4053370533.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.305.4053370533.swift)  
[typo_mage_kotlin--canary.305.4053370533.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.305.4053370533.kt)  
[typo_mage_react-native--canary.305.4053370533.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.305.4053370533.ts)  
[typo_plasma_ios-swift--canary.305.4053370533.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.305.4053370533.swift)  
[typo_plasma_kotlin--canary.305.4053370533.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.305.4053370533.kt)  
[typo_plasma_react-native--canary.305.4053370533.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.305.4053370533.ts)  
[typo_ruler_ios-swift--canary.305.4053370533.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.305.4053370533.swift)  
[typo_ruler_kotlin--canary.305.4053370533.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.305.4053370533.kt)  
[typo_ruler_react-native--canary.305.4053370533.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.305.4053370533.ts)  
[typo_sage_ios-swift--canary.305.4053370533.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.305.4053370533.swift)  
[typo_sage_kotlin--canary.305.4053370533.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.305.4053370533.kt)  
[typo_sage_react-native--canary.305.4053370533.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.305.4053370533.ts)  
[typo_sbermarket_ios-swift--canary.305.4053370533.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.305.4053370533.swift)  
[typo_sbermarket_kotlin--canary.305.4053370533.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.305.4053370533.kt)  
[typo_sbermarket_react-native--canary.305.4053370533.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.305.4053370533.ts)  
[typo_soulmate_ios-swift--canary.305.4053370533.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.305.4053370533.swift)  
[typo_soulmate_kotlin--canary.305.4053370533.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.305.4053370533.kt)  
[typo_soulmate_react-native--canary.305.4053370533.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.305.4053370533.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.129.1-canary.305.4053370533.0
  npm install @salutejs/plasma-core@1.90.1-canary.305.4053370533.0
  npm install @salutejs/plasma-icons@1.118.1-canary.305.4053370533.0
  npm install @salutejs/plasma-temple@1.126.1-canary.305.4053370533.0
  npm install @salutejs/plasma-tokens@1.32.1-canary.305.4053370533.0
  npm install @salutejs/plasma-typo@0.29.1-canary.305.4053370533.0
  npm install @salutejs/plasma-ui@1.159.1-canary.305.4053370533.0
  npm install @salutejs/plasma-web@1.156.1-canary.305.4053370533.0
  npm install @salutejs/plasma-cy-utils@0.39.1-canary.305.4053370533.0
  npm install @salutejs/plasma-sb-utils@0.88.1-canary.305.4053370533.0
  npm install @salutejs/plasma-tokens-android@2.39.1-canary.305.4053370533.0
  npm install @salutejs/plasma-tokens-ios-swift@2.39.1-canary.305.4053370533.0
  # or 
  yarn add @salutejs/plasma-b2c@1.129.1-canary.305.4053370533.0
  yarn add @salutejs/plasma-core@1.90.1-canary.305.4053370533.0
  yarn add @salutejs/plasma-icons@1.118.1-canary.305.4053370533.0
  yarn add @salutejs/plasma-temple@1.126.1-canary.305.4053370533.0
  yarn add @salutejs/plasma-tokens@1.32.1-canary.305.4053370533.0
  yarn add @salutejs/plasma-typo@0.29.1-canary.305.4053370533.0
  yarn add @salutejs/plasma-ui@1.159.1-canary.305.4053370533.0
  yarn add @salutejs/plasma-web@1.156.1-canary.305.4053370533.0
  yarn add @salutejs/plasma-cy-utils@0.39.1-canary.305.4053370533.0
  yarn add @salutejs/plasma-sb-utils@0.88.1-canary.305.4053370533.0
  yarn add @salutejs/plasma-tokens-android@2.39.1-canary.305.4053370533.0
  yarn add @salutejs/plasma-tokens-ios-swift@2.39.1-canary.305.4053370533.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-305/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-305/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-305/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-305/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-305/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-305/temple-storybook/

File: packages/plasma-tokens/lib/generator/formatters/typoKotlin.ts [hunk 1]
```diff
@@ -55,14 +86,15 @@ private val ${baseFontFamilyName} = FontFamily(
 private val ${BASE_FONT_NAME} = TextStyle(
     fontWeight = FontWeight.Normal,
     fontFamily = ${baseFontFamilyName},
-    color = lightPalette.textPrimary,
+    // lightColors.textPrimary
```
Comment (Yeti-or) [inline packages/plasma-tokens/lib/generator/formatters/typoKotlin.ts:89]: а почему сразу hex?

File: packages/plasma-typo/src/archetypes/sbermarket.ts [hunk 2]
```diff
@@ -470,7 +470,7 @@ export const typoCommonProps = (displayFontFamily: string, textFontFamily: strin
     },
     'body-xxs': {
         'font-family': textFontFamily,
-        'letter-spacing': '0.02em',
+        'letter-spacing': 'normal',
```
Comment (Yeti-or) [inline packages/plasma-typo/src/archetypes/sbermarket.ts:473]: это более корректный letter-spacing в итоге? не надо уменьшать?

**Review by Yeti-or — APPROVED** [review]

**Review by Yakutoc — APPROVED** [review]

File: packages/plasma-tokens/lib/generator/formatters/typoKotlin.ts (hunk 1 above)
Comment (neretin-trike) [reply packages/plasma-tokens/lib/generator/formatters/typoKotlin.ts:89]: hex? Не совсем понял о чём ты

File: packages/plasma-typo/src/archetypes/sbermarket.ts (hunk 2 above)
Comment (neretin-trike) [reply packages/plasma-typo/src/archetypes/sbermarket.ts:473]: ну вот да, на который я поменял, более корректный

**Pull request merged**